docs/content/_navigation.json

```diff
@@ -1159,13 +1159,13 @@
           {
             "title": "Experimental features",
             "children": [
-              {
```

Comment:
deprecated, so dropping

Reply:
not a huge deal, but ideally would do this in a separate PR where we also drop the mdx and python files